### PR TITLE
test: banner

### PR DESCRIPTION
**Pull request by DE7924**

### CODE REVIEW

File: test/src/components/banner/banner_test.dart [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+                  leadingIcon: Icons.info,
+                  trailing: const ZetaIconButton(icon: Icons.close),
+                  type: type,
+                );
+              },
+            ),
+          ),
+        );
+
+        await expectLater(tester, meetsGuideline(textContrastGuideline));
```
Comment (thelukewalton) [inline test/src/components/banner/banner_test.dart:51]: this is excellent

File: test/src/components/banner/golden/banner_negative.png
Comment (thelukewalton) [inline test/src/components/banner/golden/banner_negative.png:1]: Perhaps this is not an issue, but the screenshots make it look like nothing is aligned correctly, and the trailing button looks wrong too.

**Review by thelukewalton — COMMENTED** [review]
Should we consider getting the code coverage higher before merging this? (Currently at 87.8%)

File: test/src/components/banner/golden/banner_negative.png
Comment (DE7924) [reply test/src/components/banner/golden/banner_negative.png:1]: I've changed the iconbutton to a ZetaIcon as that is what is being using in trailing in widgetbook

File: test/src/components/banner/banner_test.dart (hunk 1 above)
Comment (DE7924) [reply test/src/components/banner/banner_test.dart:51]: Much appreciated 👍

**Review by thelukewalton — APPROVED** [review]

**Pull request merged**